editor/plugin_config_dialog.cpp

```diff
@@ -77,16 +77,19 @@ void PluginConfigDialog::_on_confirmed() {
 
 		if (lang_name == GDScriptLanguage::get_singleton()->get_name()) {
 			// Hard-coded GDScript template to keep usability until we use script templates.
-			Ref<GDScript> gdscript = memnew(GDScript);
+			Ref<Script> gdscript = memnew(GDScript);
```

Comment:
Why cast to `Script`? `GDScript` was correct, no?

Reply:
`make_template()` requires a pure `Script`.